.github/workflows/release-announcements.yml

```
@@ -14,6 +14,11 @@ jobs:
         run: |
           echo "topics=$(curl -s https://nf-co.re/pipelines.json | jq -r '.remote_workflows[] | select(.full_name == "${{ github.repository }}") | .topics[]' | awk '{print "#"$0}' | tr '\n' ' ')" | sed 's/-//g' >> $GITHUB_OUTPUT
 
+      - name: get description
+        id: get_topics
```

this is an error in the template
```suggestion
        id: get_description
```